### PR TITLE
Expanded on guideline 5.4 (inputs, outputs)

### LINKED ISSUE #3: Use input and output decorators needs some rationale
I agree with this point.  However it would be nice to say why.

Maybe something like:

# prefer input and output decorators over their @component properties

Declaring inputs and outputs as decorated members allows you to be explicit about their types.

### PR DESCRIPTION
**Pull request by SethDavenport**
Fixes #3

### CODE REVIEW

**Comment by SethDavenport** [comment]
@bertrandk

File: README.md [hunk 1]
```diff
@@ -394,7 +394,38 @@ class Button {}
...
+})
+class MyComponent {
+  @Input() foo: number;
+  @Input() bar: string;
+  @Output() quux: EventEmitter<string>;
+}
+```
+
+* It's DRY-er: you no longer have to declare the 'input-ness' and the member
+variable in two different places.
```
Comment (bertrandk) [inline README.md]: I think this is on a newline by accident.

**Comment by SethDavenport** [comment]
80-character line habit.

Fixed.

Although I will say that ultra-long markdown lines make diffs really annoying.

**Comment by bertrandk** [comment]
Thanks 😄

**Pull request merged**